### ISSUE TITLE
Trip recording without saving

### ISSUE THREAD

**Issue by mpatas** [osmandapp/OsmAnd#17909]
In some cases it would be convenient to have pretty much all the features of trip recording without actually saving that track. For example, when one is walking, running or driving a bicycle for health reasons, one might be interested in duration of the trip and its length at various points of the trip ("Should I go further, or is that enough?"), but not the actual track (especially if the memory is already close to full). Maybe that would be possible?

Of course, an obvious workaround is to delete the saved file after the trip, but that is somewhat annoying.

**Comment by DmitryAlexei** [comment]
Probably *Online tracking* option (can be found in trip recording settings https://osmand.net/docs/user/plugins/trip-recording/#recording-settings) will be good for you?

**Comment by mpatas** [comment]
That's an interesting idea; I haven't thought about it.

Still, while the documentation says "each point is transmitted to a specified URL", it does not say that this happens "instead of saving into file" (if that's what is meant to happen, documentation should be clarified). And the UI also seems to indicate that points are transmitted to that URL in addition to saving them into the file (or, to be more precise, it does not indicate the contrary). Furthermore, I have tried turning that option on, and still found a file that would have to be deleted.

I have looked at OsmAnd/src/net/osmand/plus/monitoring/LiveMonitoringHelper.java, and I didn't find a place that would turn off saving to a file (although, admittedly, I didn't look very closely).
